src/components/search-bar/search-bar.tsx

```diff
@@ -5,27 +5,37 @@ import { useText } from "../../core/utils/text";
 
 export interface SearchBarProps {
   getInputProps: UseComboboxPropGetters<unknown>["getInputProps"];
+  getLabelProps: UseComboboxPropGetters<unknown>["getLabelProps"];
 }
 
-const SearchBar: React.FC<SearchBarProps> = ({ getInputProps }) => {
+const SearchBar: React.FC<SearchBarProps> = ({
+  getInputProps,
+  getLabelProps
+}) => {
   const t = useText();
   return (
     <>
-      {/* The downshift combobox uses prop spreading by design */}
+      {/* The downshift combobox uses prop spreading by design & associated control is desctructured too */}
+      {/* eslint-disable-next-line jsx-a11y/label-has-associated-control, react/jsx-props-no-spreading */}
+      <label className="hide-visually" {...getLabelProps()}>
+        {t("searchHeaderInputLabel")}
+      </label>
       {/* eslint-disable react/jsx-props-no-spreading */}
       <input
         name="q"
         className="header__menu-search-input text-body-medium-regular"
         type="text"
         autoComplete="off"
         placeholder={t("inputPlaceholderText")}
+        aria-label={t("inputPlaceholderText")}
         {...getInputProps()}
       />
       {/* eslint-enable react/jsx-props-no-spreading */}
       <input
         type="image"
         src={searchIcon}
-        alt={t("altText")}
+        alt={t("searchHeaderIconAltText")}
+        aria-label={t("searchHeaderIconAltText")}
```

Comment:
why both alt and aria-label?

Reply:
Because the system text should describe what the search icon does/is, which I think is precise enough to use for screen readers too. 
But I'm open to suggestions of course :)

Reply:
alt texts are picked up by the screen reader, so if the texts are the same both shouldnt be necessary

Reply:
Aaaah perfect! Didn't know that! Thanks @sinejespersen 💯 
I'm correcting it now.